### PR TITLE
Enrollment Verification Process

### PR DESCRIPTION
**Pull request by nathanielbuck**
## Description
This refactors the Verify Enrollment page to get the verification and forward/back navigation and state saving working. Also, make various content fixes.

### CODE REVIEW

File: src/applications/enrollment-verification/components/EnrollmentVerificationMonth.jsx [hunk 1]
```diff
@@ -7,31 +7,31 @@ import { formatReadableMonthYear } from '../helpers';
 
 const verifiedMonthStatusMessage = (
   <p className="ev-enrollment-month_message">
-    <i className="fa fa-check-circle vads-u-color--green" /> You verified this
-    month
+    <i className="fa fa-check-circle vads-u-color--green vads-u-margin-right--1" />{' '}
```
Comment (timwright12) [inline src/applications/enrollment-verification/components/EnrollmentVerificationMonth.jsx:10]:
```suggestion
    <i className="fa fa-check-circle vads-u-color--green vads-u-margin-right--1" aria-hidden="true"/>{' '}
```

File: src/applications/enrollment-verification/components/EnrollmentVerificationMonth.jsx [hunk 2]
```diff
@@ -7,31 +7,31 @@ import { formatReadableMonthYear } from '../helpers';
...
+      haven’t verified this month
     </p>
     <VerifyYourEnrollments />
   </>
 );
 const needToVerifyMonthStatusMessage = (
   <>
     <p className="ev-enrollment-month_message">
-      <i className="fas fa-exclamation-circle vads-u-color--secondary-dark" />{' '}
+      <i className="fas fa-exclamation-circle vads-u-color--secondary-dark vads-u-margin-right--1" />{' '}
```
Comment (timwright12) [inline src/applications/enrollment-verification/components/EnrollmentVerificationMonth.jsx:26]:
```suggestion
      <i className="fas fa-exclamation-circle vads-u-color--secondary-dark vads-u-margin-right--1" aria-hidden="true"/>{' '}
```

File: src/applications/enrollment-verification/components/EnrollmentVerificationMonth.jsx [hunk 3]
```diff
@@ -7,31 +7,31 @@ import { formatReadableMonthYear } from '../helpers';
...
       You need to verify this month
     </p>
     <VerifyYourEnrollments />
   </>
 );
 const contactScoMonthStatusMessage = (
   <p className="ev-enrollment-month_message">
-    <i className="fas fa-exclamation-circle vads-u-color--secondary-dark" />{' '}
+    <i className="fas fa-exclamation-circle vads-u-color--secondary-dark vads-u-margin-right--1" />{' '}
```
Comment (timwright12) [inline src/applications/enrollment-verification/components/EnrollmentVerificationMonth.jsx:34]:
```suggestion
    <i className="fas fa-exclamation-circle vads-u-color--secondary-dark vads-u-margin-right--1" aria-hidden="true" />{' '}
```

**Review by timwright12 — CHANGES_REQUESTED** [review]

File: src/applications/enrollment-verification/sass/enrollment-verification.scss [hunk 4]
```diff
@@ -36,3 +36,8 @@
 	display: block;
 	padding: 0 0 1em;
 }
+
+// Verify your enrollments page
+.ev-verify-alert {
+	margin: 1em 0 2em;
```
Comment (timwright12) [inline src/applications/enrollment-verification/sass/enrollment-verification.scss:42]: Can we use design system classes for this? https://design.va.gov/utilities/margins

**Comment by nathanielbuck** [comment]
@timwright12, thanks for the review. I have made the changes you suggested.

**Review by FernandoBoza — APPROVED** [review]

**Review by timwright12 — APPROVED** [review]

**Pull request merged**